raiden/messages.py

```diff
@@ -488,16 +492,41 @@ class MediatedTransfer(LockedTransfer):
 
     cmdid = messages.MEDIATEDTRANSFER
 
-    def __init__(self, identifier, nonce, token, transferred_amount, recipient,
-                 locksroot, lock, target, initiator, fee=0):
+    def __init__(
+            self,
+            identifier,
+            nonce,
+            token,
+            transferred_amount,
+            recipient,
+            locksroot,
+            lock,
+            target,
+            initiator,
+            fee=0):
 
-        if nonce >= 2 ** 64:
+        if 0 > nonce >= 2 ** 64:
```

Comment:
erm...that coded example above is actually bad, because it doesn't prove the out of range check wrong, but below snippet will:
```python
>>> test = lambda nonce: 0 > nonce >= 12
>>> test(-1)
False  # should be True
>>> test(0)
False
>>> test(1)
False
>>> test(12)
False  # should be True
```